ARMeilleure/CodeGen/RegisterAllocators/HybridAllocator.cs

```diff
@@ -105,7 +110,7 @@ void SetVisited(Operand local)
 
                 if (!IsVisited(local))
                 {
-                    throw new InvalidOperationException("Local was not visisted yet. Used before defined?");
+                    ThrowNotVisisted();
```

Comment:
Same here on these two.
ThrowNotVisisted -> ThrowNotVisited
```suggestion
                    ThrowNotVisited();
```